crates/bevy_ecs/src/system/system_chaining.rs

```diff
@@ -106,6 +108,15 @@ impl<SystemA: System, SystemB: System<In = SystemA::Out>> System for ChainSystem
         self.system_a.check_change_tick(change_tick);
         self.system_b.check_change_tick(change_tick);
     }
+
+    fn get_last_change_tick(&self) -> u32 {
```

Comment:
I would argue that if we can't implement these functions for every System, they aren't System functions.
That being said, it seems like chained systems should just assume a shared change tick? At the very least, _setting_ last_change_tick for both seems correct.